### DISCUSSION TITLE
active?

### DISCUSSION THREAD

**Discussion by ghost** [jlongster/absurd-sql#62] [Q&A] [answered]
Hi.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by rogerfar** [comment, marked as answer]
This project was always meant as a bit of a prototype to what is possible.

I suggest keeping an eye on the official SQLite backend implementation which supports multiple persistent storage backends:
https://sqlite.org/wasm/doc/tip/about.md
https://sqlite.org/wasm/doc/trunk/persistence.md

**Reply by ghost** [reply]
Hi rogerfar.

> This project was always meant as a bit of a prototype to what is possible.
> I suggest keeping an eye on the official SQLite backend implementation which supports multiple persistent storage backends:
> https://sqlite.org/wasm/doc/tip/about.md
> https://sqlite.org/wasm/doc/trunk/persistence.md

thank you for feedback.
